packages/config/src/files.js

```diff
@@ -28,6 +28,7 @@ const handleFiles = async function ({ config: { build, ...config }, repositoryRo
 // itself is never relative to `buildDir` since it is contained in it.
```

Comment:
This comment is explaining what properties in `REPOSITORY_RELATIVE_PROPS` and `BUILD_DIR_RELATIVE_PROPS` are relative to. Do you think you can adjust it so that it also explains what `CONFIG_RELATIVE_PROPS` is for?

Reply:
This comment applies to those three, i.e. it also applies to `CONFIG_RELATIVE_PROPS`.
I added two comments to specify the difference between `CONFIG_RELATIVE_PROPS` and `BUILD_DIR_RELATIVE_PROPS`. I also renamed them to make the intent clearer. Please let me know what you think.

Reply:
Oh, got it! Thanks for clarifying.

Reply:
Update: I refactored it to merge both `CONFIG_RELATIVE_PROPS` and `BUILD_DIR_RELATIVE_PROPS`, which should make it clearer.